ui-kit/src/components/Column/index.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import React, { HTMLAttributes } from 'react';
+import '../../sass/components/_Column.scss';
+
+type NumberAttribute =
+  | number
+  | '1'
+  | '2'
+  | '3'
+  | '4'
+  | '5'
+  | '6'
+  | '7'
+  | '8'
+  | '9'
+  | '10'
+  | '11'
+  | '12';
+type ColumnSize = boolean | 'auto' | NumberAttribute;
+
+interface ColumnProps extends HTMLAttributes<HTMLDivElement> {
+  xs?: ColumnSize;
+  sm?: ColumnSize;
+  md?: ColumnSize;
+  lg?: ColumnSize;
+  xl?: ColumnSize;
+}
+
+const sizes = ['xl' as const, 'lg' as const, 'md' as const, 'sm' as const, 'xs' as const];
+
+export default function Column({ children, ...props }: ColumnProps): JSX.Element {
```

Comment:
이건 `ref` 포워딩 안 해줘도 될까요?

Reply:
그리고 이 부분은 컴파일러가 추론해줄 것 같습니다!

```suggestion
export default function Column({ children, ...props }: ColumnProps) {
```

Reply:
그리고 Column도 레이아웃 외에 다른 관심사를 포함하면 안되니 기존 HTML 사용방법을 그대로 살려줘야할 것 같아용. 이 컴포넌트에도 `as` 프로퍼티가 있어야 할 것 같은데 어떻게 생각하시나요 🙏

Reply:
`as`, `ref` 추가해두었어욤